### ISSUE TITLE
calender - sich selbst einladen können

### ISSUE THREAD

**Issue by Christian-H** [yakamara/redaxo_prozer#328]
im Moment ist es nicht möglich sich selbst einzuladen. Dies wäre aber sinnvoll, damit man sich Erinnerungen für einen selbst dann speichern kann und die anderen diese nicht sehen bzw. genervt sind.

**Comment by gharlan** [comment]
Wir sollten vorsichtig sein, irgendwelche Konstrukte einzuführen, die von CalDav so nicht vorgesehen sind.
Zumindest sollten wir dann schauen, was CalDav daraus macht. Also wenn wir den Termin dann so per CalDav ausliefern. Und wie der Termin dann nach Änderung in einem CalDav-Client zurückkommt etc.

**Comment by Christian-H** [comment]
Ja, gebe ich Dir Recht, sollte geprüft werden und mit CalDAV vereinbar sein Vielleicht gibt es auch einen andere Lösung, wie man für sich selbst zeitliche Notizen/Erinnerungen in einem Projekt erstellen kann, die  man nur selbst sieht? 
Das war für mich die naheliegende Lösung.
